coral/package.json

```diff
@@ -62,14 +62,19 @@
     "jest-environment-jsdom": "^29.2.1",
     "lint-staged": "^13.0.3",
     "lodash": "^4.17.21",
+    "msw": "^0.47.4",
     "prettier": "^2.7.1",
     "ts-jest": "^29.0.3",
     "ts-node": "^10.9.1",
     "typescript": "^4.6.4",
-    "vite": "^3.1.0"
+    "vite": "^3.1.0",
+    "whatwg-fetch": "^3.6.2"
   },
   "alex": {
     "noBinary": true,
     "profanitySureness": 1
+  },
+  "msw": {
+    "workerDirectory": "./"
```

Comment:
This PR also introduces `public/favicon.png` so could we change the `workerDirectory` into `public` and move our  `mockServiceWorker.js` there

Reply:
Having them in public will add them to the prod build (with vite), that's why I moved them.